include/Core/DB/TableManager.php

```diff
@@ -106,12 +106,15 @@ public function get_log_table_name() {
 	 * Insert log data into DB.
 	 *
 	 * @param array $data Data to be inserted.
+	 *
```

Comment:
Add a `@since` tag for the return value. That way we know that we introduced the return value for this function from v2.5.0

Reply:
Added @since tag.